src/api.js

```diff
@@ -167,33 +167,26 @@ export const getMyEntities = async () => {
   return [getEntityInfoForAddress(from), ...(await getEntities(from))];
 };
 
-export const getEntities = async (from) => {
-  try {
-    const identities = await getRanking(
-      [
-        {
-          algorithm: 'cryptoverse_tokens',
-          params: {
-            identity: from.toLowerCase(),
-            asset: ercs721.map(({ address: erc721Address }) => `ethereum:${erc721Address.toLowerCase()}`),
-          },
+export const getEntities = async (from) =>
```

Comment:
Could you check all usages of this function e.g. `getMyEntities` will fail if `getEntities` fails and even address entity will be unavailable.